tee-worker/ts-tests/integration-tests/common/common-types.ts

```diff
@@ -16,7 +15,7 @@ interface SubstrateWalletItem {
     [key: string]: KeyringPair;
 }
 interface BitcoinWalletItem {
-    [key: string]: bitcore.PrivateKey;
+    [key: string]: ECPairInterface;
```

Comment:
I'm just wondering if we can get rid of `EthersWalletItem`, `BitcoinWalletItem` and `SubstrateWalletItem`. 

We already have `Signer` interface so we can create wallet map:

```
{
    "evm": {
        "alice": Signer,
        "bob": Signer
    },
    "substrate": {
        "alice": Signer,
        "bob": Signer
    },
    "bitcoin": {
        "alice": Signer,
        "bob": Signer
    }
}
```

What do you think @0xverin ? Could be done in separate PR. Probably we should have Solana signer as well...

Reply:
>Could be done in separate PR. Probably we should have Solana signer as well

will create a new pr and then I think I still need to check if there are similar redundant codes in other places.